cmake/GTClangSetupCUDA.cmake

```diff
@@ -0,0 +1,60 @@
+macro( gtclang_add_nvcc_flags )
+  foreach( flag ${ARGV} )
+    set( CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} ${flag} )
+  endforeach()
+endmacro()
+
+macro( gtclang_setup_CUDA )
+
+  find_package( CUDA 8.0 REQUIRED )
+  message( STATUS "CUDA detected: " ${CUDA_VERSION} )
```

Comment:
Should we not follow the logic we use [here](https://github.com/MeteoSwiss-APN/gtclang/blob/4f54256e0468d88b4482fcebe8fd38abbade87cf/CMakeLists.txt#L77) and use the make_config_string

Reply:
unfortunately that macro is only in dawn at the moment. For a global reporting of all the dependencies found... Would be good idea to move it to mchbuild and incorporate it to gtclang. 
But maybe we can do it in a different PR